cookie-banner-rules-list.json

```diff
@@ -6498,6 +6913,271 @@
         "optOut": "#c-bns button.grey",
         "presence": "#cm"
       }
+    },
+    {
+      "id": "2f4e1235-a360-46ca-bf26-8b09645ee3d5",
+      "domains": [
+        "blablacar.com.br",
+        "blablacar.com.tr",
+        "blablacar.com.ua",
+        "blablacar.in",
+        "blablacar.mx",
+        "blablacar.rs",
+        "blablacar.ru"
+      ],
+      "cookies": {
+        "optIn": [
+          {
+            "name": "_cookies_v2",
+            "value": "1"
+          }
+        ]
+      }
+    },
+    {
+      "id": "1f0a3536-5b09-426a-b6e6-b902c556cb8a",
+      "domains": ["credit-agricole.com"],
+      "cookies": {
+        "optOut": [
+          {
+            "name": "cookie_manager",
+            "value": "!atinternet=false!recaptcha=false!addthis=false!linkedin=false!twitter=false!ezplatform=true!youtube=false!hidebanner=true"
+          }
+        ]
+      },
+      "click": {
+        "presence": "#tarteaucitronAlertBig",
+        "optOut": ".deny",
+        "optIn": ".accept"
+      }
+    },
+    {
+      "id": "1e12c729-34ff-4aa5-9317-d19309affd2c",
+      "domains": ["credit-agricole.de"],
+      "cookies": {
+        "optOut": [
+          {
+            "name": "cookie_manager",
+            "value": "!atinternet=false!recaptcha=false!addthis=false!twitter=false!ezplatform=true!youtube=false!hidebanner=true"
+          }
+        ]
+      },
+      "click": {
+        "presence": "#tarteaucitronWelcome",
+        "optOut": ".deny",
+        "optIn": ".allow"
+      }
+    },
+    {
+      "id": "f643be2a-b805-4ab0-9e67-929419e5c7c7",
+      "domains": ["credit-agricole.fr"],
+      "cookies": {
+        "optOut": [
+          {
+            "name": "TC_PRIVACY",
+            "value": "1%40006%7C86%7C3315%40%40%402524608000000%2C2524608000000%2C2524608000000%40"
+          }
+        ]
+      },
+      "click": {
+        "presence": "#tc-privacy-wrapper",
+        "optOut": "#popin_tc_privacy_button_3",
+        "optIn": "#popin_tc_privacy_button_2"
+      }
+    },
+    {
+      "id": "1944a25e-6f16-434d-8c59-0493ba587fe7",
+      "domains": ["credit-agricole.it"],
+      "cookies": {
+        "optOut": [
+          {
+            "name": "cookie_manager_cookie_marketing_enabled",
+            "value": "false"
+          },
+          {
+            "name": "cookie_manager_cookie_necessary_enabled",
+            "value": "true"
+          },
+          {
+            "name": "cookie_manager_cookie_statistic_enabled",
+            "value": "false"
+          },
+          {
+            "name": "cookie_manager_policy_accepted",
+            "value": "true"
+          }
+        ]
+      }
+    },
+    {
+      "id": "5050fb9c-fd8a-44c2-817c-23d95f494190",
+      "domains": ["credit-agricole.pl"],
+      "cookies": {
+        "optOut": [
+          {
+            "name": "ppms_privacy_7bd68a14-fbc6-44f1-9d8b-1cf0cf5eb0bb",
```

Comment:
May I ask someone to check if this ID here is the same for them? I have the impression that it is either unique and permanently assigned to a given website (in this case `credit-agricole.pl`), or it changes when they update the web page.

Reply:
in cases, like this where any change in cookie config would change the hash that cookie has as a value to reject cookies, we would prefer not to add them at all and just stick with the click rule